### PR TITLE
reset underwater state when changing team

### PR DESCRIPTION
**Pull request by bmorel**
Fixes one element in #2035, that is, changing teams removes the bar.

### CODE REVIEW

**Review by DolceTriade — APPROVED** [review]
Hopefully the next changes will be to fix this properly with netcode :D

**Comment by bmorel** [comment]
>Hopefully the next changes will be to fix this properly with netcode 

I wonder if that's actually necessary.

**Comment by DolceTriade** [comment]
Does this fix the case of spawning underwater?

**Comment by bmorel** [comment]
no.

File: src/cgame/cg_event.cpp [hunk 1]
```diff
@@ -1432,3 +1430,10 @@ void CG_CheckEvents( centity_t *cent )
 		cent->currentState.event = oldEvent;
 	}
 }
+
+// enables or disables the oxygen bar HUD element
+void CG_SetOxybar( bool underwater )
```
[the remark this answers is not in the document]
Comment (bmorel) [reply src/cgame/cg_event.cpp:1435]: OTOH "deep space" would also show the oxygen bar :)

[the remark this answers is not in the document]
Comment (DolceTriade) [reply src/cgame/cg_event.cpp:1435]: I certainly hope you don't abuse cg.underWater and cg.underwaterTime for "deep space" endevours.

[the remark this answers is not in the document]
Comment (bmorel) [reply src/cgame/cg_event.cpp:1446]: good point

**Pull request merged**